### PR TITLE
fix(@desktop/chat): GIF selector UI adjustments

### PR DESCRIPTION
**Pull request by micieslak**
### What does the PR do

Fixes: #6848 

- missing "Gif" image added
- no error message when tenor is disabled
- background dimmed-out when request dialog is opened
- gifs button highlighted when popup is opened
- positions and margins adjusted
- Search Tenor -> Search fixed
- emojis/stickers/gifs buttons order adjusted to design
- minor improvements/refactor

### Affected areas

StatusChatInput, StatusGifPopup

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

https://user-images.githubusercontent.com/20650004/184847832-8b0e6043-464b-4244-bb9d-a5f4b7c8e4ba.mp4

### CODE REVIEW

File: ui/imports/shared/status/StatusGifPopup.qml [hunk 1]
```diff
@@ -73,125 +74,144 @@ Popup {
...
+        ColumnLayout {
+            anchors.fill: parent
+            spacing: 0
+
+            Item {
+                property int headerMargin: 8
+
+                id: gifHeader
+                Layout.fillWidth: true
+                height: searchBox.height + gifHeader.headerMargin
```
Comment (caybro) [inline ui/imports/shared/status/StatusGifPopup.qml:97]: This won't work, you are inside a Layout

File: ui/imports/shared/status/StatusGifPopup.qml [hunk 2]
```diff
@@ -73,125 +74,144 @@ Popup {
...
         anchors.centerIn: parent
-        height: 290
-        width: 280
+        height: 278
+        width: 291
+
+        leftPadding: 6
+        rightPadding: 6
+        topPadding: 32
+        bottomPadding: 32
```
Comment (caybro) [inline ui/imports/shared/status/StatusGifPopup.qml:212]: horizontal/verticalPadding

File: ui/imports/shared/status/StatusGifPopup.qml [hunk 3]
```diff
@@ -209,44 +229,49 @@ Popup {
...
+        }
 
-            StyledText {
-                id: title
-                anchors.horizontalCenter: parent.horizontalCenter
-                text: qsTr("Enable Tenor GIFs?")
-                font.weight: Font.Medium
-                font.pixelSize: Style.current.primaryTextFontSize
-            }
+        StyledText {
```
Comment (caybro) [inline ui/imports/shared/status/StatusGifPopup.qml:249]: These are obsolete, pls use `StatusBaseText`

**Review by caybro — CHANGES_REQUESTED** [review]
Some minor things, otherwise good job!

File: ui/imports/shared/status/StatusGifPopup.qml (hunk 1 above)
Comment (micieslak) [reply ui/imports/shared/status/StatusGifPopup.qml:97]: not sure what won't work here. Everything seems to work as expected. ?

Comment (caybro) [reply ui/imports/shared/status/StatusGifPopup.qml:97]: Well specifying `height` inside a Layout doesn't work (if it does that's just by chance). You should use `Layout.preferredHeight` instead

Comment (micieslak) [reply ui/imports/shared/status/StatusGifPopup.qml]: Good point, thanks!

**Review by caybro — APPROVED** [review]

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**